### PR TITLE
no conflict lodash

### PR DESCRIPTION
**Pull request by benseligman**
https://crowdtap.atlassian.net/browse/CT-4192

Tested in Header: https://github.com/crowdtap/crowdtap.header/pull/522

### CODE REVIEW

File: lib/enhance.js [hunk 1]
```diff
@@ -3,7 +3,8 @@
   var Enhance, _;
 
   _ = require('./lodash-ext');
-  var lodashExt = _.noConflict();
+
+  _.noConflict();
```
Comment (armyofgnomes) [inline lib/enhance.js]: Don't you have to set it to something to use it with noconflict?

File: build/enhance.js [hunk 2]
```diff
@@ -5,6 +5,8 @@
 
   _ = require('./lodash-ext');
 
+  _.noConflict();
```
Comment (armyofgnomes) [inline build/enhance.js]: You have to set it equal to something to use it with noConflict

File: lib/enhance.js (hunk 1 above)
Comment (benseligman) [inline lib/enhance.js]: nah it puts itself on the window but the assignment on line 5 is local because coffeescript

File: src/lib/enhance.coffee [hunk 3]
```diff
@@ -1,4 +1,5 @@
 _ = require('./lodash-ext')
+_.noConflict()
```
Comment (lydiastepanek) [inline src/lib/enhance.coffee]: good catch

**Comment by lydiastepanek** [comment]
lgtm

**Comment by armyofgnomes** [comment]
lgtm

**Pull request closed without merging**